### DISCUSSION TITLE
Indici annuali

### DISCUSSION THREAD

**Discussion by annamaraspin** [italia/ansc#871] [Q&A] [answered]
Buongiorno,
domanda di semplice curiosità per il futuro.
Ho visto che con la 1.28 in ambiente webapp avete rilasciato la funzionalità di richiesta di indice annuale.
Sarà creato un servizio anche per i gestionali o sarà possibile richiedere solo tramite webapp?
grazue

**Comment by mttfranci** [comment, marked as answer]
Salve @annamaraspin, al momento è previsto che sia possibile chiedere l'indice annuale solo da web app.

Saluti

Assistenza ANSC
